app/models/volunteer_op.rb

```diff
@@ -14,7 +14,6 @@ class VolunteerOp < ActiveRecord::Base
   scope :reachskills,          -> { where(source: 'reachskills') }
   scope :remote_only,          -> { where.not(source: 'local') }
 
```

Comment:
Well personally I prefer to see real code changes in git diff instead of sorting, I agree that sorting could be helpful but in case of new feature implementation let's keep unecessary changes to the minimum to have more focus on the implementation :)